Makefile

```diff
@@ -11,6 +11,12 @@ PKGS = $(shell go list ./... | grep -v /vendor/)
 gofmtcheck:
 	@go fmt $(PKGS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi     
 
+doccheck:
+	grep -hoE '(kube_[^ |]+)' Documentation/* | sort -u > documented_metrics
```

Comment:
👍  @chlunde . 

Agree with @brancz . More details about this target especially when it fails should be great for contributors to learn and make a quick correction.